src/_includes/_version-menu-items.html

```diff
@@ -1,5 +1,5 @@
 {% assign dart1status = 'archive' %}
-{% assign dart2status = 'beta' %}
+{% assign dart2status = '' %}
```

Comment:
Ah yes, missed that, thanks! Beta should be changed to `stable`.

Reply:
done